packages/mesos/patches.json

```diff
@@ -0,0 +1 @@
+{ "kind": "git", "git": "https://github.com/mesosphere/mesos", "patches": [ { "ref": "9a9b267ad7029514a2973884db4059290a9c5f12", "description": "Set LIBPROCESS_IP into docker container." }, { "ref": "1573ffcbcf77d2aaa4b9d113904060aeb6a0cd71", "description": "Mesos UI: updated the URL generation to make it work with adminrouter." }, { "ref": "94b17b5674a06ba1d48a16a4829e4f3586b8f60c", "description": "Revert \"Fixed the broken metrics information of master in WebUI.\"" }, { "ref": "9947459906351d0026a04961b7064bf8d4951d1b", "description": "Updated mesos containerizer to ignore GPU isolator creation failure." }, { "ref": "fba8ae668411e0c84556a201eef12083187348e8", "description": "Displayed resource provider resources in GET_AGENTS response." } ] }
```

Comment:
Let's ignore the non-pretty patches.json for now. I have fixed the script so the next bump will be better.